sky.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Mourya Baruah",
+			"oem": "Redmi",
+			"device": "Redmi 12 5G",
+                        "filename": "crDroidAndroid-14.0-20240309-sky-v10.2.zip",
+                        "download": "https://sourceforge.net/projects/crdroid/files/sky/10.x/crDroidAndroid-14.0-20240309-sky-v10.2.zip/download",
+                        "timestamp": 1709971261,
+                        "md5": "5786502b96c49ff0c7a3af6f2c203c0e",
+                        "sha256": "1bc181f91114e72741a2ca9933a650a6b9591709ca62abea879ca75a57d9e9ee",
+                        "size": 1143554928,
+                        "version": "10.2",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-14-0-crdroid-10-2-redmi-12-5g-poco-m6-pro-5g-redmi-12r-5g-sky.4660956/",
+			"gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-U/07-Mar-2024/NikGapps-basic-arm64-14-20240307-signed.zip/download",
+			"firmware": "https://xiaomifirmwareupdater.com/archive/firmware/sky/",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "",
+			"paypal": "",
+			"telegram": "https://t.me/mourya_sky_support",
+			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_sky",
+			"common-dt": "",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_xiaomi_sky-kernel"
```

Comment:
renamed repo to standard https://github.com/crdroidandroid/android_kernel_xiaomi_sky
also missing 14.0 branch